en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-jdbc-user-store.md

```diff
@@ -1,6 +1,15 @@
 # Configuring a JDBC User Store
 
-JDBC user stores can be configured using the <APIM_HOME>/repository/conf/deployment.toml file's [user_store] configuration section. The user core connects to two databases (the same database is utilized by default):
+User management functionality is provided by default in WSO2 API Manager and it can be configured in the 
+`<API-M_HOME>/repository/conf/deployment.toml` file. The changes done in the `deployment.toml` file will be automatically populated to the `<API-M_HOME>/repository/conf/user-mgt.xml` file as well. 
```

Comment:
IMO, we do not need mention about the old configuration file - `user-mgt.xml`. Shall we only mention about deployment.toml?

Reply:
Removed the sections about the old configuration file `user-mgt.xml`  and updated the section to use this config to verify the configs are correctly populated or not.

https://github.com/wso2/docs-apim/pull/2174/commits/c21f3d0bd012d6cbf0be75733e110326ae5e53f3